### PR TITLE
converted profile manager into signup

### PR DESCRIPTION
**Pull request by dnascimb**
Changed the workflow a little so that you are accessing this page from the perspective of a new user at the login form, instead of after already logging into the system.

### CODE REVIEW

File: templates/signup.html [hunk 1]
```diff
@@ -77,7 +77,7 @@ <h2 class="sub-header">Personal Profile Manager</h2>
 			<div class="form-group">
 			  <label class="col-md-4 control-label" for="inputStreetAddress2">Street Address 2</label>  
 			  <div class="col-md-4">
-			  <input id="inputStreetAddress2" name="inputStreetAddress2" type="text" placeholder="Street Address 2" class="form-control input-md" required>
+			  <input id="inputStreetAddress2" name="inputStreetAddress2" type="text" placeholder="Street Address 2" class="form-control input-md">
```
Comment (dnascimb) [inline templates/signup.html:80]: found during testing - this field shouldn't be required

File: templates/index.html [hunk 2]
```diff
@@ -43,7 +43,11 @@ <h2 class="form-signin-heading">Please sign in</h2>
           </label>
         </div>
         <button class="btn btn-lg btn-primary btn-block" type="submit">Sign in</button>
+        <a class="btn btn-medium btn-link btn-block" href="{{ url_for('new_user_request') }}">New User?</a>
       </form>
+    {% for message in get_flashed_messages() %}
+      <div class="alert alert-success">{{ message }}</div>
```
Comment (dnascimb) [inline templates/index.html:49]: Place to let the user know their signup was successful. Later we can write functionality to send them an email.

File: templates/homepage.html [hunk 3]
```diff
@@ -39,13 +39,18 @@
...
-          <a class="navbar-brand" href="#">MSC</a>
+          {% if session.logged_in %}
+          <a class="navbar-brand" href="/home">MSC</a>
+          {% else %}
+          <a class="navbar-brand" href="/">MSC</a>
+          {% endif %}
         </div>
         <div id="navbar" class="navbar-collapse collapse">
           <ul class="nav navbar-nav navbar-right">
+            {% if session.logged_in %}
```
Comment (dnascimb) [inline templates/homepage.html:50]: I should have done this before... dynamically render these based on whether you are logged in or not.

File: templates/signup.html (hunk 1 above)
Comment (dnascimb) [inline templates/signup.html:80]: we also should add a "Company" field later

**Comment by edcomstock** [comment]
Yeah, originally I was going to create a profile view and it morphed into
an add.  I'll update it for view and change from within the app.

**Pull request merged**